testing/cloud_layouts/script.sh

```diff
@@ -322,6 +322,10 @@ function run-test() {
     change_deckhouse_image "${SWITCH_TO_IMAGE_TAG}" || return $?
     wait_cluster_ready || return $?
   fi
+
+  if [[ "$PROVIDER" == "Static" ]]; then
+    run_linstor_tests || return $?
+  fi
```

Comment:
Please move this peace of code inside `wait_cluster_ready` function before upmeter probes.

Reply:
done